packages/playwright/types/test.d.ts

```diff
@@ -1293,6 +1293,23 @@ interface TestConfig<TestArgs = {}, WorkerArgs = {}> {
    */
   outputDir?: string;
 
+  /**
+   * Whether to populate [metadata](https://playwright.dev/docs/api/class-testconfig#test-config-output-metadata) with Git info.
+   *
+   * **Usage**
+   *
+   * ```js
+   * // playwright.config.ts
+   * import { defineConfig } from '@playwright/test';
+   *
+   * export default defineConfig({
+   *   populateGitInfo: !!process.env.CI,
+   * });
+   * ```
+   *
+   */
+  populateGitInfo?: boolean;
```

Comment:
This must be declared in `docs/src/test-api/class-testconfig.md` instead. The APIs and docs at playwright.dev are generated from those .md files